### ISSUE TITLE
Give unique image name

### ISSUE THREAD

**Issue by arey** [spring-petclinic/spring-petclinic-cloud#14]
I would like to publish the docker images to the `springcommunity` organization: https://hub.docker.com/u/springcommunity. So people could test k8s deployment without building the docker images.
I didn't try but I suppose it will publish image without  prefix (ie: springcommunity:vets-service)

To avoid naming conflict, could we add as prefix the name of the github project: `spring-petclinic-cloud` ? Thus we will have `springcommunity:spring-petclinic-cloud-vets-service`
What do you think about?

**Comment by arey** [comment]
@odedia what do you think about?

**Comment by odedia** [comment]
Not sure I’m following, do you currently have access to this repo on dockerhub?
Usually the part after the : is the version / tag. 
So a better approach is something like 
springcommunity/spring-petclinic-cloud-vets-service:k8sv1 
Or something like that.

**Comment by arey** [comment]
Yes, I've created the  `springcommunity` organization on DockerHub. I've just added you as a member.
If neeeded, we could create another organization on DockerHub, Amazon ECR or whatever.
Your approach to concatenate the maven profile with the version could do the job. I see an issue: the `latest` tag could  reference mix profiles: `springcommunity/spring-petclinic-cloud-vets-service:k8sv1` and `springcommunity/spring-petclinic-cloud-pets-service:cloudv1` (by the way, do we have to rename the `cloud` profile to `tanzu`?)

**Comment by odedia** [comment]
It's not a best practice to deploy images with the latest tag anyway, since it will never actually pull it a second time without forcing a delete and reapply. So it's best to define the deployment yaml with a dynamic tag as well, maybe based on a build number.

**Comment by arey** [comment]
Tools like `Jib` or the Spotify `docker-maven-plugin` used on `spring-petclinic-cloud` are automatically using the Maven version to tag the docker image. It's really practical.
See https://github.com/spring-petclinic/spring-framework-petclinic#publishing-a-docker-image and
![image](https://user-images.githubusercontent.com/838318/116357174-7d41e600-a7fc-11eb-8398-fda615e9a5c4.png)

We could move on to the more recent `spring-boot:build-image` goal based on buildpacks (see https://www.baeldung.com/spring-boot-docker-images).

Whatever the solution, we have to find a way to customize the version/tag.
I propose to use a dash: `springcommunity/<service name>:<maven version>-<maven profile>`
Exemple
- `springcommunity/spring-petclinic-cloud-vets-service:2.3.1-k8s`
- `springcommunity/spring-petclinic-cloud-vets-service:2.3.1-tanzu`

**Comment by patsevanton** [comment]
I hope that will progress on this issue.

**Comment by patsevanton** [comment]
Hello!
Where i can ask question: What is repository ready for install to kubernetes? Thanks!

**Comment by arey** [comment]
@patsevanton could you try `docker pull springcommunity/spring-petclinic-cloud-discovery-service` ?
See https://github.com/spring-petclinic/spring-petclinic-cloud/pull/28

**Comment by patsevanton** [comment]
Yes. I can pull `docker pull springcommunity/spring-petclinic-cloud-discovery-service`

How can i help for this issue ?

**Comment by arey** [comment]
@patsevanton could test to deploy the application into k8s with the image I've pushed into the `springcommunity` container repository?

**Comment by patsevanton** [comment]
Thanks for answer. Now testing by skaffold. I will test later.
